### PR TITLE
Linux xhost instructions

### PR DESCRIPTION
**Pull request by dafeder**
I had to do a couple extra steps to make this work for me on linux. Not sure if this is exactly right but it's what worked for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### Linux
+
+You may need to set up access control for the X server for this to work. Install the xhost package (one is available for all distros) and run:
+
+```sh
+export DISPLAY=:0.0
```
Comment (rfay) [inline README.md:57]: Please try this and see if it works for you:
```suggestion
export DISPLAY=:0
```

I think you only need the `:0.0` with multiple screens, https://unix.stackexchange.com/a/16816/53208

File: README.md [hunk 2]
```diff
@@ -49,6 +49,15 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
...
+xhost +local:docker
```
Comment (rfay) [inline README.md:58]: I think `xhost +` should do everything, enabling permissions everywhere. Are you sure you need `xhost +local:docker` ?  Please try this out:

```suggestion
xhost +
```

**Review by rfay — CHANGES_REQUESTED** [review]
Suggestions, I'm no expert, but I think you have perhaps over-complicated it.

File: README.md (hunk 1 above)
Comment (tyler36) [reply README.md:57]: Confirming that `export DISPLAY=:0` works for WSL Ubuntu on Win11.

**Comment by dafeder** [comment]
Thanks for feedback, will test and report back!

File: README.md (hunk 1 above)
Comment (bserem) [reply README.md:57]: I can confirm that `export DISPLAY=:0` works on Linux **with X11** and a multiscreen environment.
It does **not** work with **Wayland**

File: README.md (hunk 2 above)
Comment (bserem) [reply README.md:58]: Running `xhost +` after setting `DISPLAY` was **not** necessary in my case, under Linux.
I did run `xhost` (without `+`) though. I will verify if this is required in the coming days.

Comment (tyler36) [reply README.md:58]: @bserem Looking forward to your feedback so we can continue moving forward.

**Comment by dafeder** [comment]
Apologies but the project I was working on this for was put aside and I haven't been back to this. I'm happy to re-test something if needed.

**Review by rfay — APPROVED** [review]
This should be perfect.

**Comment by bserem** [comment]
Visiting this to mention that `xhost +` is needed for Cypress tests to run under Wayland.
Our documentation is mostly ok, posting here for people who might land here while searching.

**Pull request merged**